drivers/sensor/lsm6dsv16x/lsm6dsv16x.c

```diff
@@ -938,8 +939,11 @@ static int lsm6dsv16x_init(const struct device *dev)
 	.accel_range = DT_INST_PROP(inst, accel_range),			\
 	.gyro_odr = DT_INST_PROP(inst, gyro_odr),			\
 	.gyro_range = DT_INST_PROP(inst, gyro_range),			\
-	.drdy_pulsed = DT_INST_PROP(inst, drdy_pulsed),                 \
-	LSM6DSV16X_CFG_IRQ(inst)
+	COND_CODE_1(DT_INST_NODE_HAS_PROP(inst, int1_gpios),		\
+		    (LSM6DSV16X_CFG_IRQ(inst)),				\
+		    (COND_CODE_1(DT_INST_NODE_HAS_PROP(inst, int2_gpios),	\
+				 (LSM6DSV16X_CFG_IRQ(inst)),		\
+				 ())))
```

Comment:
Wha about ?
```suggestion
	COND_CODE_1((DT_INST_NODE_HAS_PROP(inst, int1_gpios) ||	\
		     DT_INST_NODE_HAS_PROP(inst, int2_gpios)),	\
		    (LSM6DSV16X_CFG_IRQ(inst)))
```

Reply:
Ah, you can also use `IF_ENABLED`, then you don't have to have an empty "else" case

```c
IF_ENABLED((DT_INST_NODE_HAS_PROP(inst, int1_gpios) ||  \
            DT_INST_NODE_HAS_PROP(inst, int2_gpios)),   \
           (LSM6DSV16X_CFG_IRQ(inst))                   \
```

Reply:
OK, I'll take a look tomorrow to that suggestions and test them out. They both look great!

Reply:
@erwango @fabiobaltieri 
Not able to make it working with '||' in either way. Not able to find any example to...

Reply:
I tried to encapsulate the conditions in (), but it still complains .

Reply:
sorry, macro magic, can you try `UTIL_OR`?

Reply:
Yeah, I'm looking deeper into that. Also to UTIL_OR

Reply:
> sorry, macro magic, can you try `UTIL_OR`?

Yep, it works!